docs/API_SPECS_DEVICE_PROPERTIES.md

```diff
@@ -0,0 +1,42 @@
+API specifications - Device Properties
+======================================
+
+## Get a list of device properties
+
+The list of device properties contains properties from all device models.
+
+Request:
+```
+GET /v1/deviceproperties
```

Comment:
Should this be an API based on DeviceModel instead?

like... GET /v1/devicemodels/propertiesList?

I also believe the "list" terminology is consistent with our naming scheme for other APIs.

Reply:
done